src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIDE = 100;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public final Circle getDefaultFigure() {
+        return new Circle(10, Color.WHITE);
+    }
+
+    public Square getRandomSquare() {
```

Comment:
```suggestion
    private Square getRandomSquare() {
```

All methods that create figure, should be private
private methods should be placed after all public methods